### PR TITLE
feat: New Relic integration

### LINKED ISSUE #878: feat: New Relic integration
Hi all :wave: 

We already released the addon-framework where we want to provide out-of the box integrations with third party tools. It would be super helpful if someone familiar with New Relic could assist on building the New Relic integration. The idea is to inform New Relic about changes via [New Relic Custom events](https://docs.newrelic.com/docs/telemetry-data-platform/ingest-apis/introduction-event-api/). This can be very helpful when analyzing issues in New Relic. 

We have already written the integration for Datadog, and relevant code for this is:
- [datadog.js](https://github.com/Unleash/unleash/blob/master/src/lib/addons/datadog.js)
- [datadog-definition.js](https://github.com/Unleash/unleash/blob/master/src/lib/addons/datadog-definition.js)

Documentation:
https://unleash.github.io/docs/addons/datadog

### PR DESCRIPTION
**Pull request by alexandrusavin**
## About the changes
Add New Relic integration based on issue #878.

image

image

Closes #878

### CODE REVIEW

**Review by chriswk — APPROVED** [review]
Thank you for your contribution, much appreciated. I'm going to merge this.

File: src/lib/addons/new-relic.test.ts [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+        expect(jsonBody.eventType).toBe('UnleashServiceEvent');
+        expect(jsonBody.unleashEventType).toBe(event.type);
+        expect(jsonBody.featureName).toBe(event.data.name);
+        expect(jsonBody.environment).toBe(event.environment);
+        expect(jsonBody.createdBy).toBe(event.createdBy);
+        expect(jsonBody.createdByUserId).toBe(event.createdByUserId);
+        expect(jsonBody.createdAt).toBe(event.createdAt.getTime());
+
+        expect(options.body).toMatchSnapshot();
```
Comment (chriswk) [inline src/lib/addons/new-relic.test.ts:164]: I think you can drop this line, that way the test doesn't rely on confirming that a byte buffer is exactly the same each time. The rest of your test is verification enough

**Review by chriswk — CHANGES_REQUESTED** [review]

File: src/lib/addons/new-relic.test.ts (hunk 1 above)
Comment (alexandrusavin) [reply src/lib/addons/new-relic.test.ts]: Just pushed a new commit removing this line

**Review by chriswk — APPROVED** [review]
Going to approve this, and update the addons counter myself for the failing test on addons. Thank you again @alexandrusavin

**Pull request merged**